src/main/java/com/dev/cinema/service/impl/UserServiceImpl.java

```diff
@@ -0,0 +1,25 @@
+package com.dev.cinema.service.impl;
+
+import com.dev.cinema.dao.UserDao;
+import com.dev.cinema.lib.Inject;
+import com.dev.cinema.lib.Service;
+import com.dev.cinema.model.User;
+import com.dev.cinema.service.UserService;
+
+import java.util.Optional;
```

Comment:
```suggestion
import com.dev.cinema.service.UserService;
import java.util.Optional;
```

Reply:
The import order is incorrect approximately in every second class
Fix that, please